docs/background/case-studies/heritage-impact-assessment.md

```diff
@@ -0,0 +1,489 @@
+---
+title: Heritage Impact Assessment
+---
+
+# Heritage Impact Assessment using QField
+
+## From Qgis to Qfield and Vice Versa: How the New Android Application Is Facilitating the Work of the Archaeologist in the Field
+
+**Roberto Montagnetti 1,* and Giuseppe Guarino 2**
+
+<sub><sup>1 adArte s.r.l. Archeologia, Restauro, ICT, 47921 Rimini, Italy</sup></sub>

```

Comment:
I guess the `<sub><sup>` is here on purpose?

Reply:
It's to make the text smaller. Do you have a better solution?

Reply:
`<small>` would be an option, but if you have put it here on purpose, that's also ok. Just wanted to be sure it's not a leftover.

Reply:
Please use footnotes like here https://opengisch.github.io/QField-docs/background/case-studies/vanilla-survey/

See https://squidfunk.github.io/mkdocs-material/reference/footnotes/#usage for usage